draft-ietf-quic-http.md

```diff
@@ -1638,16 +1645,27 @@ H3_FRAME_UNEXPECTED.
 
 # Error Handling {#errors}
 
-QUIC allows the application to abruptly terminate (reset) individual streams or
-the entire connection; see Sections 2.4 and 5.3 of {{QUIC-TRANSPORT}}.  These
-are referred to as "stream errors" or "connection errors" (see Section 11 of
-{{QUIC-TRANSPORT}}) and have associated error codes, but do not necessarily
-indicate a problem with the connection or either implementation.  For example, a
-stream can be reset if the requested resource is no longer needed.
+When a stream cannot be completed successfully, QUIC allows the application to
+abruptly terminate (reset) that stream and communicate a reason; see Section 2.4
+of {{QUIC-TRANSPORT}}. This is referred to as a "stream error."  Stream errors
+are distinct from HTTP status codes which indicate error conditions.  Stream
+errors indicate that the sender did not transfer the full request or response,
+while HTTP status codes indicate the result of a request that was successfully
+received.
+
+If an entire connection needs to be terminated, QUIC similarly provides
+mechanisms to communicate a reason; see Section 5.3 of {{QUIC-TRANSPORT}}.  This
+is referred to as a "connection error."
+
+Both stream errors and connection errors have associated error codes; see
+Section 11 of {{QUIC-TRANSPORT}}.  However, they do not necessarily indicate a
```

Comment:
They are called error codes here, but refered to as "error of type H3_..." throughout the document.
If "error code" and "error type" are indeed the same concept, please consider unifying your terminology, e.g., changing "error of type H3_..." to "error using error code H3_...".

Reply:
I think there's a better alternative. In the paragraph that introduces stream errors, state that there can be multiple variants of "reason", descrived an error type, which has an associated wire encoding error code. Do the same for the para that introduces connection errors.

Then delete the first sentence of this para and possibly merge it with the next one.

Reply:
@LPardue, I've taken part of your idea and tried to use it to improve the terminology.  PTAL.